index.html

```diff
@@ -5,57 +5,208 @@
+            or sign-ups required.
+          </p>
+          <div class="primary">
+            <ul>
+              <li>HTML</li>
+              <li>CSS</li>
+              <li>JavaScript</li>
+            </ul>
+          </div>
+          <button class="btn btn-primary" type="button">See Project</button>
+        </div>
+      </div>
+      <div class="title grid">
+        <div>
+          <img src="./assets/snapshot4.png" alt="portfolio"/>
+        </div>
+        <div class="textTitle">
+          <div class="text">
+            <div class="skill">
+              <h4>Multi-Post Stories</h4>
+
+              <ul class="ul">
+                <li class="first-child">CANOPY</li>
+                <li>Back End Dev</li>
+                <li>2015</li>
+              </ul>
+            </div>
+          </div>
+          <p class="texts">
+            A daily selection of privately personalized reads; no accounts
+            or sign-ups required.
+          </p>
+          <div class="primary">
+            <ul>
+              <li>HTML</li>
+              <li>CSS</li>
+              <li>JavaScript</li>
+            </ul>
+          </div>
+          <button class="btn btn-primary" type="button">See Project</button>
+        </div>
+      </div>
+    </section>
+    <section class="about">
+      <div class="container">
+        <div>
+          <h2 class="about_header">About<br />Myself</h2>
+          <p class="details">
+            Hello I’m a software developer! I can help you build a product ,
+            feature or website Look through some of my work and experience!
+            If you like what you see and have a project you need coded,
+            don’t hestiate to contact me.
+          </p>
+        </div>
+        <div>
+          <h2 class="media_header">LET'S CONNECT</h2>
+          <ul class="media_ul">
+            <li class="media_li">
+              <a href="#" target="_blank">
+                <img src="./assets/gitHubIcon.svg" alt="GitHub icon"/>
+              </a>
+            </li>
+            <li class="media_li">
+              <a href="#" target="_blank">
+                <img src="./assets/linkedInIcon.png" alt="LinkedIn icon"/>
+              </a>
+            </li>
+            <li class="media_li">
+              <a href="#" target="_blank">
+                <img src="./assets/Vector.png" alt="Vector icon"/>
+              </a>
+            </li>
+            <li class="media_li">
+              <a href="#" target="_blank">
+                <img src="./assets/twitterIcon.svg" alt="Twitter icon"/>
+              </a>
+            </li>
+            <li class="media_li">
+              <a href="#" target="_blank">
+                <img src="./assets/mIcon.svg" alt="M_Icon"/>
+              </a>
+            </li>
+          </ul>
+        </div>
+      </div> 
+      <a href="#" class="btn btn-primary">Get my resume</a>
+      <div class="languages">
```

Comment:
- [ ]  Nice working on this `section` kindly re-align the frameworks and skills menu so it aligns with what you have in the Figma Template.

## Your Design
![fig-2](https://user-images.githubusercontent.com/48371156/166123438-0ca37868-5830-4ac3-87ee-a8c0262d8871.png)

## Figma Template
![fig-1](https://user-images.githubusercontent.com/48371156/166123443-8c3c9a03-af14-4914-b99b-b5208edb3969.png)